frontend/src/tabs/zoom.pug

```diff
@@ -46,15 +47,15 @@
 
   .sorting.mui-form--inline
     .mui-select
-      select.medium-font(v-model="commitsSortType")
+      select(v-model="commitsSortType")
```

Comment:
this

Reply:
This `font-size` is overwritten in the original css. The `font-weight` is **normal**. Therefore, removed it.